packages/kit/test/apps/hash-based-routing/test/test.js

```diff
@@ -72,13 +72,13 @@ test.describe('hash based navigation', () => {
 		await page.locator('a[href="/#/reroute-a"]').click();
 		await expect(page.locator('p')).toHaveText('rerouted');
 		let url = new URL(page.url());
-		expect(url.hash).toBe('#/rerouted');
+		expect(url.hash).toBe('#/reroute-a');
 
 		await page.goto('/');
 
 		await page.locator('a[href="/#/reroute-b"]').click();
 		await expect(page.locator('p')).toHaveText('rerouted');
 		url = new URL(page.url());
-		expect(url.hash).toBe('#/rerouted');
+		expect(url.hash).toBe('#/reroute-b');
```

Comment:
turns out there even was a test that could've caught this, but the test assertion was wrong